.flake8

```diff
@@ -15,6 +15,18 @@ format = pylint
 
 exclude = .git,__pycache__
 
+per-file-ignores = \
+  # Module imported but unused
```

Comment:
Do we ignore unused imports because the linter expects the imports are used in that specific file, while the imports are used indirectly in other files? It could be useful to explain that in a comment.